src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -13,6 +13,10 @@ case class ThurloeKeyValue(
   userId: Option[String] = None,
   keyValuePair: Option[FireCloudKeyValue] = None)
 
+case class ThurloeKeyValues(
+  userId: Option[String] = None,
+  keyValuePairs: Option[Seq[FireCloudKeyValue]] = None)
```

Comment:
`FireCloudKeyValue` is a bit misleading. The name makes me think it a special FireCloud object, but really it's just a key/value pair where both the key and value are optional. So, two questions, (1) would `OptionalKeyValue` be a better name and (2) why is `key` optional?